lib/services/lwm2mHandlers/registration.js

```diff
@@ -188,10 +188,20 @@ function addUnsupportedAttributes(payload, configuration, deviceInformation, cal
             if (omaRegistry[value.objectType].resources) {
```

Comment:
CNR entry missing?

Reply:
Fixed in b17a3d45cc0a778958faef8677ef349c64327064